### DISCUSSION TITLE
Which things should I learn to use the scripting feature of Trilium?

### DISCUSSION THREAD

**Discussion by WantToLearnJapanese** [TriliumNext/Trilium#2935] [Q&A] [answered]
Which things should I learn to use the scripting feature of Trilium?

Node.js? Javascript?

Since I have no experience in programming with Javascript, it will be helpful for me if you can suggest some good starting points for using Trilium with the scripting feature.

**Comment by zadam** [comment, marked as answer]
Mainly JavaScript. You don't really need to know node.js as it's largely just JavaScript + backend APIs (which are not very relevant in Trilium).

Depending on what you need to do, some familiarity with jQuery, HTML and CSS might be also needed.
